packages/account-export/package.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "@near-wallet-selector/account-export",
+  "version": "7.4.0",
+  "peerDependencies": {
+    "near-api-js": "^1.1.0"
+  }
```

Comment:
I think we should set the range of NAJ the same as in other packages:
```json
  "peerDependencies": {
    "near-api-js": "^0.44.2 || ^1.0.0"
  }
```
The reason we still need `^0.44.2` is that this is the version that was used by Wallet Selector for the majority of the time and there could be dApps that haven't moved to the latest version of NAJ yet.